### DISCUSSION TITLE
Is it possible to use parseq to create a video effect similar to loopback wave？

### DISCUSSION THREAD

**Discussion by wuyue0509** [rewbs/sd-parseq#141] [Q&A]
Is it possible to use deforum to produce a video effect similar to loopback wave, and how to control the change of seeds?

**Comment by rewbs** [comment]
Hi! Yes, there are 2 examples of this in the templates: 

<img width="804" alt="image" src="https://github.com/rewbs/sd-parseq/assets/74455/1ed88673-4917-4673-bb9d-292f4ec78bf9">

Strength wave uses a seed that changes on every frame and an oscillating strength to periodically pick up major changes from the seed. Seed wave evolves the seed in bursts to create the wave effect.

Try those out and let me know what you think.
